documentation/docs/commands/validating-configuration.md

```diff
@@ -2,19 +2,26 @@
 sidebar_position: 1
 ---
 
-# Validate configuration
+# Validating configuration
 
-Monaco validates configuration files in a directory by performing a dry run. It will check whether your Dynatrace config files are in a valid JSON format, and whether your tool configuration YAML files can be parsed and used.
+Monaco validates configuration files in a directory by performing a dry run. 
+It will check whether your Dynatrace config files are in a valid JSON format and 
+whether your tool configuration YAML files can be parsed and used.
 
-To validate the configuration, execute a `monaco dry run` on a YAML file as shown below.
+To validate the configuration, execute a `monaco --dry-run` on a YAML file as shown below.
 
-Create the file at `src/pages/my-react-page.js`:
-
-```jsx title="run monaco in dry mode"
- ./monaco -dry-run --environments=project/sub-project/my-environments.yaml
+```shell title="Validating your configuration"
+monaco --dry-run --environments=environments.yaml
 2020/06/16 16:22:30 monaco v1.0.0
```

Comment:
Could you please remove the output from this while you're at it? :) 
That would get it in line with the other cmd line samples that can actually be copied. The output is not too helpful here anyway

Reply:
Addressed :+1: